src/sentry/static/sentry/app/components/assistant/helper.jsx

```diff
@@ -80,15 +86,23 @@ const AssistantHelper = createReactClass({
                 guide={this.state.currentGuide}
                 step={this.state.currentStep}
                 onClose={closeGuide}
+                handleDismiss={this.handleDismiss}
               />
             ) : (
               <SupportDrawer onClose={this.handleSupportDrawerClose} />
             )}
           </div>
         ) : (
-          <a onClick={this.handleDrawerOpen} className="assistant-cue">
-            {cueText}
-          </a>
+          <div>
+            <a onClick={this.handleDrawerOpen} className="assistant-cue">
+              {cueText}
+            </a>
+            <a
+              onClick={this.handleDismiss}
+              className="icon-close assistant-cue"
+              hidden={!this.state.currentGuide}
```

Comment:
Since cueText and hidden are tied together (i.e. are determined by the same logic), it would be better to define a hidden variable at the beginning of `render` alongside `cueText`. Also nit: it's more common to not render the element at all if you don't want it to be there, instead of hiding it. So you'd do: `!hidden && (<a>...</a>)`